### PR TITLE
Use api_clients method to update file size of items on the server

### PR DESCRIPTION
**Pull request by paddatrapper**
This is not pretty and would really improve from an OO API Client (#1308). But it should fix some of the errors seen in #1302

### CODE REVIEW

File: python_apps/api_clients/api_clients/utils.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def __init__(self, name, url, logger=None, api_key=None):
             self.logger = logger
         self.auth = KeyAuth(api_key)
 
-    def __call__(self, _post_data=None, params=None, **kwargs):
+    def __call__(self, _post_data=None, _put_data=None, params=None, **kwargs):
```
Comment (jooola) [inline python_apps/api_clients/api_clients/utils.py:94]: Let's protect us from future breaking changes and enforce using a keyword based argument passing to this function.

```patch
-    def __call__(self, _post_data=None, _put_data=None, params=None, **kwargs):
+    def __call__(self, *, _post_data=None, _put_data=None, params=None, **kwargs):
```

Comment (paddatrapper) [reply python_apps/api_clients/api_clients/utils.py:94]: Need to check in the version1.py and version2.py files. Those and their tests should be the only things calling it

File: python_apps/pypo/pypo/pypofile.py [hunk 2]
```diff
@@ -88,8 +88,8 @@ def copy_file(self, media_item):
                 os.chmod(dst, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
 
                 if media_item["filesize"] == 0:
-                    file_size = self.report_file_size_and_md5_to_airtime(
-                        dst, media_item["id"], host, username
+                    file_size = self.report_file_size_and_md5_to_libretime(
```
Comment (jooola) [inline python_apps/pypo/pypo/pypofile.py:91]: While you are renaming the function, could we replace `libretime` with `api`, it is explicit and I think we shouldn't use the project name anyway.

**Review by jooola — COMMENTED** [review]
Haven't tested this, but looks good. Only a small comment on naming.

**Pull request merged**